src/Abstractions/NexusMods.Abstractions.Library.NexusMods/NexusModsModPageMetadata.cs

```diff
@@ -0,0 +1,36 @@
+using JetBrains.Annotations;
+using NexusMods.Abstractions.MnemonicDB.Attributes;
+using NexusMods.Abstractions.NexusWebApi.Types;
+using NexusMods.MnemonicDB.Abstractions.Attributes;
+using NexusMods.MnemonicDB.Abstractions.Models;
+
+namespace NexusMods.Abstractions.Library.NexusModsLibrary;
+
+/// <summary>
+/// Represents a remote mod page on NexusMods.
+/// </summary>
+[PublicAPI]
+public partial class NexusModsModPageMetadata : IModelDefinition
+{
+    private const string Namespace = "NexusMods.Library.NexusModsModPageMetadata";
+
+    /// <summary>
+    /// The ID of the mod page.
+    /// </summary>
+    public static readonly ModIdAttribute ModId = new(Namespace, nameof(ModId)) { IsIndexed = true };
+
+    /// <summary>
+    /// The name of the mod page.
+    /// </summary>
+    public static readonly StringAttribute Name = new(Namespace, nameof(Name));
+
+    /// <summary>
+    /// The game of the mod page.
+    /// </summary>
+    public static readonly GameDomainAttribute Game = new(Namespace, nameof(Game)) { IsIndexed = true };
```

Comment:
Use `GameDomain` as the name of the property to make it clear what it is. We have actual Game classes.